components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -128,27 +146,58 @@ export default function () {
                             })}
                             {activeWorkspaces.length > 0 && <div className="py-6"></div>}
                             {inactiveWorkspaces.length > 0 && (
-                                <div className="p-3 text-gray-400 bg-gray-50 dark:bg-gray-800 rounded-xl text-sm text-center">
-                                    Unpinned workspaces that have been inactive for more than 14 days will be
-                                    automatically deleted.{" "}
-                                    <a
-                                        className="gp-link"
-                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                <div className="pt-14">
+                                    <div className="border-t border-gray-200 dark:border-gray-800"></div>
```

Comment:
**suggestion**: The border helps, I'd suggest to remove it and find other ways to create the needed visual balance here. Reusing the same structure of the workspace entry element could help (see :a:) as this would feel like a natural continuation of the active workspaces list. In case this still feelsSince this could

| BEFORE | AFTER :a: | AFTER :b: |
|-|-|-|
| <img width="1440" alt="InactiveWorkspaces-0" src="https://user-images.githubusercontent.com/120486/172178908-0aefc1b0-f592-44fa-8868-a78f88c8d327.png"> | <img width="1440" alt="InactiveWorkspaces-a" src="https://user-images.githubusercontent.com/120486/172178932-1c80dbc0-2f4f-4c68-8490-3a20fd0b976d.png"> | <img width="1440" alt="InactiveWorkspaces-b" src="https://user-images.githubusercontent.com/120486/172178944-9910417a-9b90-4c40-99b1-3cd982c4ebf3.png"> |

Posting below some early iterations for future reference:

| Iteration 1 | Iteration 2 | Iteration 3 |
|-|-|-|
| <img width="1440" alt="InactiveWorkspaces-3" src="https://user-images.githubusercontent.com/120486/172180816-4fcb9a68-c929-4a51-adab-1b1fb31385e5.png"> | <img width="1440" alt="InactiveWorkspaces-5" src="https://user-images.githubusercontent.com/120486/172180829-aa32ebe0-10f9-4aaa-bd48-94d6951b2e4a.png"> | <img width="1440" alt="InactiveWorkspaces-a" src="https://user-images.githubusercontent.com/120486/172180861-1c821efe-50fc-4c57-9507-07068dba22de.png"> |

Reply:
What is the meaning of the circled arrow icon on the left?

Reply:
The purpose of this icon was to :a: mimic the element structure of the workspace list item to help to associate this element with the workspaces (see [principle of grouping](https://en.wikipedia.org/wiki/Principles_of_grouping)[^1]) and :b: resemble a [history icon](https://duckduckgo.com/?q=history+icon&atb=v314-1&iax=images&ia=images) to help convey the contents of this collapsible component.

However, this first iteration of the icon could be confusing due to similarities with an [undo icon](https://duckduckgo.com/?q=undo+icon&atb=v314-1&iax=images&ia=images).

Here are a few more variations we could try:

| Icon variations for the inactive workspaces element |
|-|
| <img width="1342" alt="InactiveWorkspaces" src="https://user-images.githubusercontent.com/120486/172580872-43edd6b4-153b-4273-8003-c6b99c6358a1.png"> |

| With the current history icon | With an improved history icon | 
|-|-|
| <img width="1440" alt="InactiveWorkspaces-a" src="https://user-images.githubusercontent.com/120486/172581479-11d89a63-3fc4-49ed-9531-5fecb54eb260.png"> | <img width="1440" alt="InactiveWorkspaces-1" src="https://user-images.githubusercontent.com/120486/172581981-1ab911e7-df4b-4e2c-8ad8-bd6a0cd61ff0.png"> |

Alternatively, we could simply replace the icon with an arrow icon from the first line. Dropping the icon altogether could easily create visual imbalance I'd like to avoid.

| With an arrow icon | Without an icon |
|-|-|
| <img width="1440" alt="InactiveWorkspaces-b" src="https://user-images.githubusercontent.com/120486/172581495-9ea0d40b-cfd1-4eb8-ae71-37ea93fe4972.png"> | <img width="1440" alt="InactiveWorkspaces-c" src="https://user-images.githubusercontent.com/120486/172581502-fe5ccf0c-3198-4aa8-ac73-54e3a9bfe6ba.png"> |

[^1]: Elements that are visually similar will be perceived as related.

Reply:
Marking as resolved. Moved forward with an arrow icon. ✔️